content/post/stash-v2022.05.18/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Introducing Stash v2022.05.12
+date: 2022-05-18
+weight: 10
+authors:
+  - Piyush Kanti Das
+tags:
+  - kubernetes
+  - stash
+  - backup
+  - restore
+  - docs
+  - nats
+  - elasticsearch
+---
+
+We are very excited to announce Stash `v2022.05.18`.  In this release, we have added exciting new features and improvements. We have squashed a few bugs as well. You can find the complete changelog [here](https://github.com/stashed/CHANGELOG/blob/master/releases/v2022.05.18/README.md). We are going to highlight the major changes in this post.
+
+## New Features
+
+Here, we are going to highlight the new features that have been introduced in this release.
+
+#### Add support for Elasticsearch 8.2.0
+
+In this release, we have added Elasticsearch 8.2.0 support to our Stash Elasticsearch add-on. This addition improved Stash's capability to backup and restore large Elacticsearch databases. 
+
+#### Add Nat's Account Backup Support
+
+We have added Account Backup Support to our Nat's Addon. Now, you can backup your Nat's account and Nat's stream both by Stash.
+
+## Bug Fixes and Improvements
+
+We have fixed some bugs and did some upgrade. Here are a few notable changes,
+
+- **Fix**: Fixed a bug to pass the ImagePullSecrets to the backup job properly now [#1445](https://github.com/stashed/stash/pull/1445).
```

Comment:
- **Bug Fix:** Fixed bug that caused the ImagePullSecrets from RuntimeSettings of BackupConfiguration and RestoreSession weren't passed properly to the respective backup or restore job. [#1445](https://github.com/stashed/stash/pull/1445)